connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/webhook/InboundWebhookRestController.java

```diff
@@ -111,7 +111,7 @@ private ResponseEntity<?> processWebhook(
       } else if (e instanceof WebhookConnectorException webhookConnectorException) {
         connectorResponse = handleWebhookConnectorException(webhookConnectorException);
       } else {
-        connectorResponse = ResponseEntity.internalServerError().build();
+        connectorResponse = ResponseEntity.internalServerError().body(e.getMessage());
```

Comment:
I'm not sure if this is always a good idea to return the arbitrary exception in body. I think there is a chance that we may leak sensitive information this way or expose something unwanted, as this is an endpoint open to the internet. WDYT?